HACKING.md

```diff
@@ -1,288 +1,169 @@
-# Goldstone Hacking Guide
+Goldstone Server Hacking Guide
+================
 
-This explains how to install and run Goldstone locally, so you can do code development on it.
 
-The recommended steps are:
+This document explains how to install and run Goldstone Server locally (mostly in docker containers), so you can do code development on the project.  The instructions assume a MacOS X development environment with [homebrew](http://brew.sh/) and [Virtualbox](https://www.virtualbox.org/wiki/Downloads) installed.  The setup process for other environments will be similar, but inevitably be different.
```

Comment:
Unnecessary?
"The setup process for other environments will be similar, but inevitably be different."